### ISSUE TITLE
AlphabeticallySortedUses fixer delete 'group use' statements

### ISSUE THREAD

**Issue by jonathan1055** [slevomat/coding-standard#1554]
**group use** statements are not supported by Slevomat Coding-standard, but if a file contains a group use statement it gets silently deleted when the fixer is run. For example, with the following:
```
use thirdproject\D;
use otherproject\{B, C};
use myproject\A;
```
When phpcbf is run the resulting fixed file is
```
use myproject\A;
use thirdproject\D;
```
The middle statement is deleted. This is similar to issue #1547 and if it is agreed that this is a bug and a PR would be considered then I will take a look at it.

**Comment by kukulich** [comment]
Yes, we have sniff to disallow group uses hovewer this sniff should not break the code.

**Comment by jonathan1055** [comment]
#1555 is ready for review. Because group use statements are not saved in `getUseStatementPointers()` we can't check the actual statement (like we did with multi-part use statements). But it is simple enough to re-search the file for a `T_OPEN_USE_GROUP`.

One question - I don't know what type of source code file structure would provide any more than one array within `$getFileUseStatements`. If this is a possibility, and there are cases where there are two (or more) sets of use statements that need to be sorted, can you give me an example? The new search for `T_OPEN_USE_GROUP` may need to be restricted within a smaller range of pointers. Currently it just searches the whole file.
